### DISCUSSION TITLE
Convert yaml to environment variable declaration

### DISCUSSION THREAD

**Discussion by 1Mark** [mikefarah/yq#1532] [Q&A]
```
- +name: a
  +value: b
- +name: c
  +value: d
```

Given this yaml how can i convert this to

`export A=B; C=D;`

strenv looks relevant but im unsure

**Comment by mikefarah** [comment]
You can just concatenate the strings together like so:

```
yq '.[] | ("export " + .+name + "=" + .+value)' file.yaml
```

Explanation:
- `.[]` splats the array into its component items
- `("export " + .+name + "=" + .+value)` builds the desired string from each item

**Reply by 1Mark** [reply]
@mikefarah thank you! is there a way to ensure it's all on one line ?
